src/rocker/dockerclient/dockerclient.go

```diff
@@ -48,7 +50,8 @@ type Config struct {
 func NewConfig() *Config {
 	certPath := os.Getenv("DOCKER_CERT_PATH")
 	if certPath == "" {
-		certPath = "~/.docker"
+		usr, _ := user.Current()
```

Comment:
this is not cross platform API, the build can pass with CGO, but at runtime it will throw error on some platform.
see also: http://stackoverflow.com/questions/7922270/obtain-users-home-directory